pkg/agent/flowexporter/connections/deny_connections_test.go

```diff
@@ -71,58 +75,50 @@ func TestDenyConnectionStore_AddOrUpdateConn(t *testing.T) {
 	mockIfaceStore.EXPECT().GetInterfaceByIP(tuple.SourceAddress.String()).Return(nil, false)
 	mockIfaceStore.EXPECT().GetInterfaceByIP(tuple.DestinationAddress.String()).Return(nil, false)
 
-	denyConnStore := NewDenyConnectionStore(mockIfaceStore, mockProxier, testStaleConnectionTimeout)
+	pq := priorityqueue.NewExpirePriorityQueue(testActiveFlowTimeout, testIdleFlowTimeout)
+	denyConnStore := NewDenyConnectionStore(mockIfaceStore, mockProxier, pq, testStaleConnectionTimeout)
 
 	denyConnStore.AddOrUpdateConn(&testFlow, refTime.Add(-(time.Second * 20)), uint64(60))
 	expConn := testFlow
 	expConn.DestinationServicePortName = servicePortName.String()
 	actualConn, ok := denyConnStore.GetConnByKey(flowexporter.NewConnectionKey(&testFlow))
 	assert.Equal(t, ok, true, "deny connection should be there in deny connection store")
 	assert.Equal(t, expConn, *actualConn, "deny connections should be equal")
+	assert.Equal(t, 1, pq.Len(), "Length of the expire priority queue should be 1")
 	checkDenyConnectionMetrics(t, len(denyConnStore.connections))
 
 	denyConnStore.AddOrUpdateConn(&testFlow, refTime.Add(-(time.Second * 10)), uint64(60))
 	expConn.OriginalBytes = uint64(120)
-	expConn.DeltaBytes = uint64(120)
 	expConn.OriginalPackets = uint64(2)
-	expConn.DeltaPackets = uint64(2)
 	expConn.StopTime = refTime.Add(-(time.Second * 10))
 	actualConn, ok = denyConnStore.GetConnByKey(flowexporter.NewConnectionKey(&testFlow))
 	assert.Equal(t, ok, true, "deny connection should be there in deny connection store")
 	assert.Equal(t, expConn, *actualConn, "deny connections should be equal")
+	assert.True(t, actualConn.IsActive)
+	assert.Equal(t, 1, pq.Len())
 	checkDenyConnectionMetrics(t, len(denyConnStore.connections))
 }
 
-func TestDenyConnectionStore_DeleteConnWithoutLock(t *testing.T) {
+func TestDenyConnectionStore_(t *testing.T) {
```

Comment:
incomplete name?

Reply:
Thanks for catching that. That's resulted from refactoring. I have moved the `deleteConnWithoutLock` method to connection.go, didn't realized there's a testing function for that. Have now moved the unit test case to connections_test.go.